.eslintrc.js

```diff
@@ -18,6 +18,35 @@ module.exports = {
   },
   rules: {
     'babel/no-invalid-this': 1,
+    'consistent-return': [2],
+    'dot-notation': [2],
+    'func-names': [2],
+    'jsx-a11y/label-has-for': [
+      'error',
+      {
+        components: ['label'],
+        required: {
+          some: ['nesting', 'id'],
+        },
+        allowChildren: true,
+      },
+    ],
+    'no-console': [1],
+    'no-implicit-coercion': [2],
```

Comment:
Lets keep the `!!` idiom. Way terser than `Boolean` constructor

```suggestion
    'no-implicit-coercion': [2, { "allow": ["!!"] } ],
```

Reply:
👍

Reply:
done